src/functionalTest/java/uk/gov/hmcts/reform/sscs/functional/TransformationFunctionalTest.java

```diff
@@ -170,4 +170,21 @@ public void transform_appeal_created_case_when_missing_office_maternity_allowanc
 
         verifyResponseIsExpected(expectedJson, transformExceptionRequest(jsonRequest, OK.value()));
     }
+
+    @Test
+    public void sscs2_transform_appeal_created_case_when_all_fields_entered() throws IOException {
+        String expectedJson = getJson("exception/output/sscs2_expected_all_fields_entered.json");
+
+        String person1Nino = generateRandomNino();
+        String person2Nino = generateRandomNino();
```

Comment:
You can't have a person2 on SSCS2 so might be worth removing from the test so it is more realistic

Reply:
ok removed